RecoBTag/SecondaryVertex/python/candidateBoostedDoubleSecondaryVertexAK8Computer_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+candidateBoostedDoubleSecondaryVertexAK8Computer = cms.ESProducer("CandidateBoostedDoubleSecondaryVertexESProducer",
+    beta = cms.double(1.0),
+    R0 = cms.double(0.8),
+    maxSVDeltaRToJet = cms.double(0.7),
+    weightFile = cms.FileInPath('RecoBTag/SecondaryVertex/data/BoostedDoubleSV_CA15_BDT.weights.xml.gz')
```

Comment:
The idea is to use the same C++ code for both AK8 and CA15 jets but the AK8 training we had as an external was still the old one so this was a temporary workaround (the CA15 training already uses the latest set of variables). Once the externals get updated, we will also update the code (hopefully, this can all happen before the final 750 release).